libs/etterlatte-behandling-model/src/main/kotlin/no/nav/etterlatte/libs/common/omregning/OpprettOmregningResponse.kt

```diff
@@ -0,0 +1,6 @@
+package no.nav.etterlatte.libs.common.omregning
+
+import no.nav.etterlatte.libs.common.behandling.SakType
+import java.util.UUID
+
+data class OpprettOmregningResponse(val behandlingId: UUID, val forrigeBehandlingId: UUID, val sakType: SakType)
```

Comment:
Sliter litt med begrepene her, er omregning forskjellig fra regulering eller revurdering?

Reply:
Omregning er eit samleomgrep for der noko har endra seg i verda utanfor, som gjer at vi må rekne om kor mykje vi skal betale ut, men vilkårsvurderinga vår er uendra.

Regulering er eit (kanskje det fremste) eksempelet på det, vil tru endringar i avkorting på OMS er eit anna.